packages/site/src/components/PaymentChannelContainer.tsx

```diff
@@ -38,7 +47,17 @@ export default function PaymentChannelContainer({
         focusedPaymentChannel={focusedPaymentChannel}
         setFocusedPaymentChannel={setFocusedPaymentChannel}
       />
-      <PaymentChannelDetails paymentChannel={focusedPaymentChannel} />
+      <div className={classes.paymentDetails}>
+        <PaymentChannelDetails
```

Comment:
We don't want to mock anything here if possible. We can use storybook for that.

We need to get all of the data from the props or from the rpc client as necessary.

For example, the counter party is in the channel participants array.

We also need to reflect the inbound / outbound nature of the channel properly too. @NiloCK can advise.

Reply:
Okay